<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>498</width>
    <height>523</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Générateur de mot de passe</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>251</width>
      <height>431</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="listPwdLayout">
     <item>
      <widget class="QPlainTextEdit" name="listPwd">
       <property name="readOnly">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="copyPwd">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Copier le mot de passe</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>481</width>
      <height>41</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Longueur mot de passe</string>
       </property>
       <property name="margin">
        <number>0</number>
       </property>
       <property name="indent">
        <number>-1</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="lengthPwd">
       <property name="toolTip">
        <string extracomment="Longueur du mot de passe"/>
       </property>
       <property name="minimum">
        <number>6</number>
       </property>
       <property name="maximum">
        <number>20</number>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QComboBox" name="typePwd">
       <property name="minimumSize">
        <size>
         <width>250</width>
         <height>0</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>Alphanumérique</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Hex</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>60</y>
      <width>221</width>
      <height>431</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="optionLayout">
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="page">
        <widget class="QGroupBox" name="caracterOption">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>10</y>
           <width>211</width>
           <height>111</height>
          </rect>
         </property>
         <property name="title">
          <string>Caractères options</string>
         </property>
         <widget class="QCheckBox" name="minuscule">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>191</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Minuscule (a-z)</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QCheckBox" name="majuscule">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>40</y>
            <width>191</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Majuscule (A-Z)</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QCheckBox" name="symbols">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>60</y>
            <width>191</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Symbols</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
         </widget>
         <widget class="QCheckBox" name="numerique">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>80</y>
            <width>191</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Numérique</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </widget>
        <widget class="QGroupBox" name="specialCaracter">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>129</y>
           <width>211</width>
           <height>71</height>
          </rect>
         </property>
         <property name="title">
          <string>Caractères spéciaux</string>
         </property>
         <widget class="QCheckBox" name="checkBoxCustomSet">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>21</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
         <widget class="QLineEdit" name="customSet">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>30</y>
            <width>151</width>
            <height>21</height>
           </rect>
          </property>
         </widget>
        </widget>
        <widget class="QGroupBox" name="otherOption">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>210</y>
           <width>211</width>
           <height>61</height>
          </rect>
         </property>
         <property name="title">
          <string>Autres options</string>
         </property>
         <widget class="QCheckBox" name="uniqueCaractere">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>30</y>
            <width>191</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>Unique</string>
          </property>
         </widget>
        </widget>
       </widget>
       <widget class="QWidget" name="page_2"/>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="amountPwd">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>70</height>
        </size>
       </property>
       <property name="title">
        <string>Nombre mot de passe</string>
       </property>
       <widget class="QSpinBox" name="amount">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>30</y>
          <width>61</width>
          <height>24</height>
         </rect>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>30</number>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Nombre</string>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="genPwd">
       <property name="text">
        <string>Générer</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>498</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuA_propos">
    <property name="title">
     <string>A propos</string>
    </property>
    <addaction name="actionInfos"/>
   </widget>
   <addaction name="menuA_propos"/>
  </widget>
  <action name="actionSd">
   <property name="text">
    <string>sd</string>
   </property>
  </action>
  <action name="actionInfos">
   <property name="text">
    <string>Générateur de mot de passe</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
